runtimebp/signal.go

```diff
@@ -0,0 +1,44 @@
+package runtimebp
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+// ShutdownHandler is the callback type used in HandleSignals.
+type ShutdownHandler func(signal os.Signal)
+
+var defaultSignals = []os.Signal{
+	// Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods
+	syscall.SIGTERM,
```

Comment:
Should we also handle `SIGINT` by default? 

https://github.com/reddit/baseplate.py/blob/3638ffd8f561aa90f16e126acb167a41cc7b1727/baseplate/server/__init__.py#L210

Reply:
Added.

Reply:
Didn't we just remove `os.Interrupt` from the previous discussion? https://github.com/reddit/baseplate.go/pull/142#pullrequestreview-393311225

Reply:
Yes. But we changed `os.Kill` into `syscall.SIGTERM` (`os.Kill` is 9 while `SIGTERM` is 15). Interrupt is still useful when developers run the service locally, in the foreground to do some local debugging (so they can use ^C to stop it).

Reply:
Thanks for the clarification :)